app/src/main/java/org/mozilla/focus/biometrics/BiometricAuthenticationDialogFragment.kt

```diff
@@ -80,11 +85,17 @@ class BiometricAuthenticationDialogFragment : AppCompatDialogFragment(), Lifecyc
     interface BiometricAuthenticationListener {
         fun onCreateNewSession()
```

Comment:
Can we rename this its not clear its involving biometric and not our sessionmanager sessions